packages/xdl/src/detach/AndroidShellApp.js

```diff
@@ -1040,24 +1041,83 @@ export async function runShellAppModificationsAsync(
   );
 }
 
-async function buildShellAppAsync(context: StandaloneContext, sdkVersion: string) {
+async function buildShellAppAsync(
+  context: StandaloneContext,
+  sdkVersion: string,
+  buildType: string
+) {
   let shellPath = shellPathForContext(context);
+  const ext = buildType === 'app-bundle' ? 'aab' : 'apk';
 
-  if (context.build.android) {
-    let androidBuildConfiguration = context.build.android;
+  const isRelease = !!context.build.android;
+  // concat on those strings is not very readable, but only alternative here is huge if statement
+  const debugOrRelease = isRelease ? 'Release' : 'Debug';
```

Comment:
This could be called `buildType` https://developer.android.com/studio/build/build-variants

Reply:
Then `buildType` would have to become productType? resultType?